src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilerTests.cs

```diff
@@ -167,7 +228,15 @@ private void TestWorkItemFiler(SarifLog sarifLog, SarifWorkItemContext context)
             int expectedWorkItemsCount = context.GetProperty(ExpectedWorkItemsCount);
 
             createWorkItemCalled.Should().Be(expectedWorkItemsCount);
-            createAttachmentCalled.Should().Be(expectedWorkItemsCount);
+
+            if (adoClient == true)
```

Comment:
>if (adoClient == true) [](start = 12, length = 22)

suggest reducing this to a single statement:

createAttachmentCount.Should().Be(adoClient ? expectedWorkItemsCount : 0);

Reply:
Oh, I really like this.

---
In reply to: [409067203](https://github.com/microsoft/sarif-sdk/pull/1852#discussion_r409067203) [](ancestors = 409067203)